src/oic/oauth2/provider.py

```diff
@@ -153,9 +154,14 @@ class Provider(object):
     endp = [AuthorizationEndpoint, TokenEndpoint]
 
     def __init__(self, name, sdb, cdb, authn_broker, authz, client_authn,
-                 symkey="", urlmap=None, iv=0, default_scope="",
+                 symkey=None, urlmap=None, iv=0, default_scope="",
                  ca_bundle=None, verify_ssl=True, default_acr="",
                  baseurl='', server_cls=Server, client_cert=None):
+
+        if symkey is not None and symkey == "":
```

Comment:
Maybe we can postpone this check till the point where the symkey is actually needed? Otherwise this check seems a little bit pointles since it passes for `None` which will break stuff.